lib/services/plan_payment_service.ts

```diff
@@ -0,0 +1,23 @@
+import { Stripe } from 'stripe';
+
+export interface PlanPaymentRequestBody {
```

Comment:
Please follow the naming conventions on click types repo.
```suggestion
export interface CreatePlanPaymentRequestBody {
```

Reply:
I updated